igcollect/sentry.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""Sentry stat collector
+
+Copyright (c) 2022 InnoGames GmbH
+"""
+from argparse import ArgumentParser
+from subprocess import run, PIPE
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='sentry.')
+    parser.add_argument('--config_path', default='/etc/sentry/sentry.conf.py')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    proc = run(
```

Comment:
A small check if the run succeeded would be nice. In case the config is broken etc. At least to make script exit with a warning instead of crash.